Formula/loki.rb

```diff
@@ -3,7 +3,7 @@ class Loki < Formula
   homepage "https://grafana.com/loki"
   url "https://github.com/grafana/loki/archive/v2.2.1.tar.gz"
   sha256 "4801a9418c913bcca5e597d09f0f7ce1f5a7ce879f8dba3e8fe86057cb592bcf"
-  license "Apache-2.0"
+  license "AGPL-3.0-or-later"
```

Comment:
recently changed: https://github.com/grafana/loki/commit/a9d85de4aa5290cf2f8b2dca5d08645bbd0dc66c

Reply:
dropped this, checksum also changed